en/collect/add-entry-using-an-id.md

```diff
@@ -60,6 +60,12 @@ The [International Association for Cryptologic Research](https://www.iacr.org/)
 
 ID search is carried out using the Cryptology ePrint ID.
 
+### JSTOR
+
+Jstor is a digital library of academic content in many disciplines.
```

Comment:
You could take the same description of Jstor as in https://docs.jabref.org/collect/import-using-online-bibliographic-database

Reply:
Please, put the 2 sentences in different paragraphs, so as to reproduce the layout used for the other search engines.